app/models/edition.rb

```diff
@@ -242,6 +242,10 @@ def substitute
     )
   end
 
+  def gone?
+    schema_name == "gone" || document_type == "gone" || rendering_app.nil?
```

Comment:
is this an accurate reflection of the nature of "gone"? if so, perhaps we should create a data sanitisation task; i'm not convinced three different types of goneness is a good idea.